src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -152,5 +153,12 @@ public AddressBook load() throws StorageOperationException {
     public String getPath() {
         return path.toString();
     }
+    
+    public void fileExist() throws FileNotFoundException{
```

Comment:
1. you need a better name for the method since you are checking and the result of this is a boolean value.
2. Throwing an exception from this method seems like an overkill. A `boolean` check is more suitable?